policykit/policyengine/views.py

```diff
@@ -611,25 +612,96 @@ def clean_up_proposals(action, executed):
 
 @csrf_exempt
 def initialize_starterkit(request):
-    from policyengine.models import StarterKit, CommunityPlatform
-
-    starterkit_name = request.POST['starterkit']
-    community_name = request.POST['community_name']
-    creator_token = request.POST['creator_token']
-    platform = request.POST['platform']
-
-    starter_kit = StarterKit.objects.get(name=starterkit_name, platform=platform)
-
-    community = CommunityPlatform.objects.get(community_name=community_name)
-    starter_kit.init_kit(community, creator_token)
-
-    logger.info('starterkit initialized')
-
-    redirect_route = request.GET.get("redirect")
-    if redirect_route:
-        return redirect(f"{redirect_route}?success=true")
```

Comment:
This `redirect` parameter was used when a user installs a second platforms from the "settings" page. For example if I'm logged in with Discord, and I want to install to my Slack too, the flow looks like this:
1. Navigate to /main/settings
2. On the top right, click Add Integration > Slack
3. Approve Slack consent screen
4. Now I see the starterkit page and select a starterkit to use with my new SlackCommunity. Click on a starterkit.
5. I am redirected back to /main/settings

That flow is broken by this change.  Ideally we can fix that before merging it in @rickygv99 @amyxzhang

Reply:
Updated!

Reply:
great nice work!!